test/import_test.erl

```diff
@@ -45,3 +45,8 @@ override_test_() ->
       [?_assertMatch(doit, override:doit()),
        ?_assertMatch([5,5], override:doit(5)),
        ?_assertMatch([doit, 5, 10], override:doit(5, 10))]}].
+
+reserved_test_() ->
+    [{<<"Use reserved words as function name">>,
```

Comment:
Perhaps "use non-alphanumeric atom as function name" ?

Reply:
I kept this test as reserved word, and added a new with the non-alphanumeric function names.
Let me know if you disagree